contributing_to_docs/doc_guidelines.adoc

```diff
@@ -421,24 +434,22 @@ The `GET` operation can be used to do something.
 Answer by typing `Yes` or `No` when prompted.
 
 |System or software variable to be replaced by the user
-a|$$`_<project>_`$$
+a|$$`<project>`$$
 
-$$`_<deployment>_`$$
+$$`<deployment>`$$
 
 a|
 Use the following command to roll back a deployment, specifying the deployment name:
 
-`oc rollback _<deployment>_`
-
-This is ONLY used when showing the command syntax using the sidebar block.
+`oc rollback <deployment>`
 
 |System or software configuration parameter or environment variable
-a|$$`*ENVIRONMENT_VARIABLE*`$$
+a|$$*`ENVIRONMENT_VARIABLE`*$$
 
-$$`*PARAMETER*`$$
-a|Use the `*IP_ADDRESS*` environment variable for the server IP address.
+$$*`PARAMETER`*$$
+a|Use the *`IP_ADDRESS*` environment variable for the server IP address.
 
-The `*MAX_PODS*` parameter limits the number of pods you can have.
+The *`MAX_PODS`* parameter limits the number of pods you can have.
```

Comment:
These are now backwards. It was correct the way it was (and how we've been using it). From http://asciidoctor.org/docs/user-manual/#mono:

"Monospaced text can be bold and italicized, as long as the markup sets are entered in the right order. The monospace markup must be the outermost set, then the bold set, and the italic markup must always be the innermost set."